### ISSUE TITLE
MultiMC crashing on loading of modpack, wondering if problem is with MultiMC or modpack

### ISSUE THREAD

**Issue by bioquantum** [MultiMC/MultiMC5#3181]
System Information
-----------------------------
MultiMC version: 0.6.11-1370

Operating System: osx64 macOS Catalina

Summary of the issue or suggestion:
----------------------------------------------
On loading of Valhelsia 2 (other modpacks have worked, but they seem to start up in a different fashion, so I'm not sure if it's a modpack or my system's problem), the Mojang start up screen comes up and loads up to about 95%, then sticks for a few minutes, and crashes.

What should happen:
------------------------------
I imagine the game should be starting

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1. Download the Valhelsia 2 zip and set it up as a new instance.

2. Launch instance.

3. Wait for crash.

Suspected cause:
---------------------------
Not sure, I'm not well versed in code. 

Logs/Screenshots:
----------------------------
 [m [32m[22:29:54] [CraftPresence-ShutDown-Handler/INFO] [STDERR/]: [com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils:shutDown:403]: java.lang.IllegalStateException: IPCClient (ID: 450485984333660181) is not connected!
 [m [32m[22:29:54] [CraftPresence-ShutDown-Handler/INFO] [STDERR/]: [com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils:shutDown:403]: 	at com.gitlab.cdagaming.craftpresence.utils.discord.rpc.IPCClient.checkConnected(IPCClient.java:384)
 [m [32m[22:29:54] [CraftPresence-ShutDown-Handler/INFO] [STDERR/]: [com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils:shutDown:403]: 	at com.gitlab.cdagaming.craftpresence.utils.discord.rpc.IPCClient.close(IPCClient.java:325)
 [m [32m[22:29:54] [CraftPresence-ShutDown-Handler/INFO] [STDERR/]: [com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils:shutDown:403]: 	at com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils.shutDown(DiscordUtils.java:401)
 [m [32m[22:29:54] [CraftPresence-ShutDown-Handler/INFO] [STDERR/]: [com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils:shutDown:403]: 	at com.gitlab.cdagaming.craftpresence.utils.discord.DiscordUtils$1.run(DiscordUtils.java:178)
 [m [32m[22:29:54] [CraftPresence-ShutDown-Handler/INFO] [craftpresence/]: Shutting Down CraftPresence...

Process exited with code 255.

Additional Info:
---------------------------
Apologies if I did this poorly, this is my first post after reading one other post. Regardless, thank you, genius saviors.

**Comment by phit** [comment]
please add your full log instructions: https://github.com/MultiMC/MultiMC5/wiki/Minecraft-Logs#uploading-logs

**Comment by QuImUfu** [comment]
Please upload the full log.  
This may be an Out of memory error, but can't say for sure without the full log. Please make sure to give this modpack at least ~4GB of ram. Forge requires ridiculous amounts of ram during startup.

**Comment by Hordeq** [comment]
Getting the same error in the same situation. It's definitely not a memory error as I've tried 4gb/8gb/16gb. Able to run it perfectly fine in VM Win10. Using Mint 20 it gives the crash error. I've tried taking mods out systematically. I'm able to do Omnia in the same way and it works fine. I'm pulling my hair out. I hope it's just something stupidly simple overlooked on my part. Just weird we both had the same issue.

[minecraft.log](https://github.com/MultiMC/MultiMC5/files/4866563/minecraft.log)

**Comment by RedstoneLP2** [comment]
This appears to be a known issue in FancyMenu. although I'm not sure if the issue in macOS is the same. Disabling it for now seems to fix it for Linux, can't try it for macOS

**Comment by Hordeq** [comment]
Removing fancymenu didn't help here in Linux.

**Comment by QuImUfu** [comment]
@Hordeq Use java 8 for modded minecraft. Almost no one tests their mods for other versions.  
FanyMenu got an update that hopefully should fix the issue in the log file. Maybe there are other issues. Post a fresh log.

**Comment by Hordeq** [comment]
I've not used Java 8 in years for any mods. But, I tried it here and without fancymenu it loaded one time a bit further then crashed and we're back to the same issue. This is the log with fancymenu removed. I also tried the fancymenu update but since it doesn't make it to the init for it.. it didn't give any different results. I started this log with a fresh copy in case I screwed something up along the line trying to fix it.

[minecraft1.log](https://github.com/MultiMC/MultiMC5/files/4869709/minecraft1.log)
